src/containers/RegisterPage/components/AgeInput.jsx

```diff
@@ -1,34 +1,59 @@
-import {useRef, useState} from "react";
+import {useState} from "react";
 import UserService from "../../../services/UserService";
 import css from "../RegisterPage.module.css";
 
 function AgeInput(props) {
-    const inputAgeRef = useRef();
+    const [isHidden, setIsHidden] = useState(false);
+    const [ageInput, setAgeInput] = useState('');
+    const [ageError, setAgeError] = useState('');
+    const [disableBtn, setDisableBtn] = useState(true);
 
-    const [isClicked, setIsClicked] = useState(false);
+    const isAgeValid = (e) => {
```

Comment:
I like the way you implemented this function. But for name I have a couple of comments. If you name functions with kinda questions - isSomething(a good name, but), it is common to expect that it will return boolean. Your function does not return a value, it handles age input validation. So maybe `ageValidation` would be good name for it.
P.S. Same for other inputs